src/components/Content/Animals/Animals.js

```diff
@@ -1,13 +1,154 @@
 import React from 'react'
+import {useState} from 'react'
 import styles from './Animals.module.css'
+import AnimalElem from './AnimalElem.js'
+import AnimalCategory from './AnimalCategory.js'
+
+
+const dogImg1 = 'https://wallpaperaccess.com/full/90604.jpg'
+const dogImg2 = 'https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F28%2F2020%2F10%2F13%2Fcorgi-dog-POPDOGNAME1020.jpg'
+const dogImg3 = 'https://pbs.twimg.com/profile_images/1351720980972933122/I3MnYUdm_400x400.jpg'
+const dogImg4 = 'https://www.cesarsway.com/wp-content/uploads/2019/10/AdobeStock_190562703.jpeg'
+const dogImg5 = 'https://i.insider.com/5484d9d1eab8ea3017b17e29?width=600&format=jpeg&auto=webp'
```

Comment:
instead of creating multiple const set this as an object or array. This object can also be referenced from a separate file of mockdata